src/Conference/ConferenceDetails.jsx

```diff
@@ -84,6 +95,14 @@ const ConferenceDetails = () => {
         <p>{city}</p>
         <p>{description}</p>
         <StyledLink to="/">Back</StyledLink>
+        <br />
+        <Button
+          onClick={() => {
+            deleteAConference(id);
```

Comment:
A small performance improvement: I'd hoist this out of the render method as every time this re-renders you create a new reference. :)